src/RecordManager/Base/Command/Records/Export.php

```diff
@@ -716,4 +795,134 @@ protected function writeRecord(string $record): void
         ++$this->currentBatchCount;
         file_put_contents($this->currentFile, $record . "\n", FILE_APPEND);
     }
+
+    /**
+     * Add an node to the XML record
+     *
+     * @param \SimpleXMLElement $xml       XML record
+     * @param array             $path      XML path
```

Comment:
Maybe add "from parseXmlPath()" to the doc comment to clarify what kind of array we're working with here.